src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  activeTodo: Todo | null,
+  onModalOpen: (todo: Todo) => void,
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  activeTodo,
+  onModalOpen,
+}) => {
+  const {
+    id,
+    title,
+    completed,
+  } = todo;
+
+  return (
+    <tr
+      data-cy="todo"
+      className={classNames({
+        'has-background-info-light': id === activeTodo?.id,
```

Comment:
DRY. Create variable for `id === activeTodo?.id` and reuse it here and in other places